main.go

```diff
@@ -71,15 +98,20 @@ func main() {
 	flag.StringVar(&backendCfg.VaultKubernetesPath, "vault.kubernetes-path", "kubernetes", "Vault kubernetes login path")
 	flag.StringVar(&watchNamespaces, "watch-namespaces", "", "Comma separated list of namespaces that secrets-manager will watch for SecretDefinitions. By default all namespaces are watched.")
 	flag.StringVar(&excludeNamespaces, "exclude-namespaces", "", "Comma separated list of namespaces that secrets-manager will not watch for SecretDefinitions. By default all namespaces are watched.")
+	opts := zap.Options{
+		Development: true,
+	}
+	opts.BindFlags(flag.CommandLine)
 	flag.Parse()
+	logger := zap.New(zap.UseFlagOptions(&opts))
+	logf.SetLogger(logger)
+	backendLog := logf.Log.WithName("backend")
 
 	if versionFlag {
 		fmt.Printf("Secrets Manager %s\n", version)
 		os.Exit(0)
 	}
 
-	logger := zap.Logger(enableDebugLog).WithName("backend")
-
```

Comment:
How do you enable debug log if you remove this?

Reply:
Debug log options re-added